metadata-ingestion-examples/mce-cli/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+  id 'org.springframework.boot'
+  id 'java'
```

Comment:
Why not use `application` plugin directly?

Reply:
Sorry, which plugin? I just followed examples from our other apps because I'm not very familiar with spring :)

Reply:
Springboot might have a different way of handling this, but normal java CLI uses this plugin: https://docs.gradle.org/current/userguide/application_plugin.html

Reply:
right, you have to use the spring plugin.